bool ConfirmNewTaskReceived
bool ConfirmContinueWork
bool ConfirmPauseWork
bool ConfirmEmergencyStop
bool ConfirmCancelLastTask
bool ConfirmStartTask

